easybuild/easyconfigs/m/MPICH/MPICH-4.1.2-GCC-11.3.0.eb

```
@@ -0,0 +1,20 @@
+name = 'MPICH'
+version = '4.1.2'
+
+homepage = 'https://www.mpich.org/'
+description = """MPICH is a high-performance and widely portable implementation 
+of the Message Passing Interface (MPI) standard (MPI-1, MPI-2 and MPI-3)."""
+
+toolchain = {'name': 'GCC', 'version': '11.3.0'}
+
+source_urls = ['https://www.mpich.org/static/downloads/%(version)s']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['3492e98adab62b597ef0d292fb2459b6123bc80070a8aa0a30be6962075a12f0']
+
+configopts = 'FFLAGS="-w -fallow-argument-mismatch -O2" --with-devices=ch4:ucx --with-ucx=$EBROOTUCX'
+
+dependencies = [
+    ('UCX', '1.13.1'),
```

```suggestion
    ('UCX', '1.12.1'),
```
For GCC 11.3.0 we are only having a easyconfig for UCX 1.12.1, c.f.: https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/u/UCX/UCX-1.12.1-GCCcore-11.3.0.eb

Alternatively you could keep UCX 1.13.1, if you bump the toolchain to GCC 12.2.0